### DISCUSSION TITLE
How to close flash effect when I set image source?

### DISCUSSION THREAD

**Discussion by wangzyon** [zauberzeug/nicegui#921] [Q&A] [answered]
### Description

I use button to change image source, Flashing effect will appear when switching,  my eyes can't bear it, can i close flash? thanks! 
```py
from nicegui import ui
import random

def imshow(image):
    image_urls = ['https://cdn.stocksnap.io/img-thumbs/960w/airplane-sky_DYPWDEEILG.jpg',
                  'https://picsum.photos/id/377/640/360',
                  'https://picsum.photos/id/565/640/360']
    image.set_source(random.choice(image_urls))

image = ui.image()

ui.button('Click me!', on_click=lambda : imshow(image))
```

**Comment by rodja** [comment]
use [`ui.interactive_image`](https://nicegui.io/documentation#interactive_image), which is build for seamless source updates. See https://github.com/zauberzeug/nicegui/blob/7deff0c433ca54d06bd857af7416e9583ccac6ca/examples/opencv_webcam/main.py#L44-L45

**Comment by falkoschindler** [comment, marked as answer]
Try the "no-transition" and/or the "no-spinner" prop:

```py
image = ui.image().props('no-transition no-spinner')
ui.button('Click me!', on_click=lambda: image.set_source(f'https://picsum.photos/id/{random.randint(1, 500)}/640/360'))
```

See https://quasar.dev/vue-components/img for more information.
